File: src/main/java/no/nav/syfo/domain/FeiletSending.java
package no.nav.syfo.domain;

import lombok.Data;
import lombok.experimental.Accessors;

import java.time.LocalDateTime;

@Data
@Accessors(fluent = true)
public class FeiletSending {
    public Long id;
    public Long oppfolgingsplanId;
    public int number_of_tries;
    public int max_retries;
    public LocalDateTime sistEndretDato;
    public LocalDateTime opprettetDato;
}

File: src/main/kotlin/no/nav/syfo/domain/FeiletSending.kt
package no.nav.syfo.domain

import java.time.LocalDateTime

const val MAX_RETRIES = 6;

data class FeiletSending(
        val id: Long,
        val oppfolgingsplanId: Long,
        val number_of_tries: Int = 1,
        val max_retries: Int = MAX_RETRIES,
        val sist_endret: LocalDateTime = LocalDateTime.now(),
        val opprettet: LocalDateTime = LocalDateTime.now()
)
